mystring.h

```diff
@@ -14,5 +14,5 @@ class String {
     ~String();
     const char* toString() const;
     friend std::ostream & operator<< (std::ostream& os, const String &s);
-    friend unsigned int stringLength(const String mystring);
+    friend unsigned int stringLength(char* const str);
```

Comment:
it's better to use it as a class member, just like "length()"

Reply:
the idea is that noone has access to the internal string apart from the class itself

Reply:
removed friend function and added class member "length()"